web/samples_index/lib/src/templates.dart

```diff
@@ -0,0 +1,219 @@
+import 'dart:convert';
+
+import 'data.dart';
+import 'util.dart' as util;
+
+String _e(String s) =>
+    HtmlEscape(HtmlEscapeMode.attribute).convert(s);
+String _ea(String s) => HtmlEscape(HtmlEscapeMode.element).convert(s);
```

Comment:
What does the "a" in `_ea` signify?

Reply:
I think these need to be renamed, originally they were _escape and _escapeAttribute I think

Reply:
fixed